app/src/main/java/com/example/mydigitalcloset/AboutUsPageActivity.java

```diff
@@ -58,14 +58,14 @@ public void onClick(View view) {
               finish();
           }
       });
-        /*binding.settingsButton.setOnClickListener(new View.OnClickListener() {
```

Comment:
check functionality before merging

Reply:
Functionality does work and I am now able to merge